web/client/utils/QueryParamsUtils.js

```diff
@@ -0,0 +1,52 @@
+import url from "url";
+import {get} from "lodash";
+
+/**
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const getRequestLoadValue = (name, state) => {
+    const search = get(state, 'router.location.search') || '';
+    const { query = {} } = url.parse(search, true) || {};
+    if (query[name]) {
+        try {
+            return JSON.parse(query[name]);
+        } catch (e) {
+            if (query[name].length) {
+                return query[name];
+            }
+            return null;
+        }
+    }
+    return null;
+};
+
+export const postRequestLoadValue = (name, storage = sessionStorage) => {
```

Comment:
Document this function (please add parameters and correct if I'm wrong)
Example: 

```suggestion
/**
 * Retrieves parameters from the `queryParams` entry (JSON) of the passed storage (by default `sessionStorage`). 
 * Example: `#/viewer/openlayers/0?center=0,0&zoom=5
 */
export const postRequestLoadValue = (name, storage = sessionStorage) => {
```